WooCommerce/Classes/ViewRelated/Products/Cells/ProductReviewsTableViewCell.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+import Gridicons
+
+
+// MARK: - ProductReviewsTableViewCell
+//
+class ProductReviewsTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var reviewLabel: UILabel!
+    @IBOutlet weak var reviewTotalsLabel: UILabel!
+    @IBOutlet weak var starRatingView: RatingView!
+
+    var starRating: Int? {
+        didSet {
+            guard let starRating = starRating else {
+                starRatingView.isHidden = true
+                return
+            }
+
+            starRatingView.rating = CGFloat(starRating)
+        }
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        reviewLabel.applyBodyStyle()
```

Comment:
I know it might sound a bit overkill, but I find this very asymmetric. We do apply styles to the labels right here, but we delegate to a separate method to configure the star. I would definitely encapsulate the styling of each label in a separate method, if only to make this more symmetric.